src/uwtools/resources/rocoto.jsonschema

```diff
@@ -0,0 +1,44 @@
+{
+    "attrs": {
+        "realtime": {
+            "type": "string",
+            "enum": [
+                "true",
+                "false",
+                "T",
+                "F",
+                "t",
+                "f",
+                "True",
+                "False",
+                "FALSE",
+                "TRUE"
+            ]
+        }
+        "scheduler": {
+            "type": "string",
+            "enum": [
+                "sge",
+                "lsf",
+                "lsfcray",
+                "ll",
+                "torque",
+                "moabtorque",
+                "moab",
+                "pbspro",
+                "slurm",
+                "cobalt"
+            ]
+        }
+    }
+    "cycledefs": {
+        "type": "string",
+        "format": "date" " " "date" " " "time"
+    }
+    "log": {
+        "type": "string"
+    }
+    "entities": {
+        "type": "string"
```

Comment:
Entities will be a dict of key/value pairs. Here the key seems to be a string. Is this the only type we would be seeing?

For example: 
```
entities:
     foo: 1
     bar: “/some/path”
```

The value pairs type doesn't seem to need to be a string. Would this be sufficient to account for all options the user could supply? Entities from the SRW documentation are required, but looking over the schema_with_metask RelaxNG they don't seem to be included in the requirement for a valid Rocoto XML.

Reply:
I don't think entities are necessarily required, per se. If they are provided under this section, they should be a dictionary of key-value pairs. I'm not sure if we want to necessarily restrict what the values could be? Perhaps only strings, integers, floats, and bools?